components/RecipeForm.js

```diff
@@ -238,44 +198,47 @@ export default function RecipeForm({ recipeToEdit }) {
     return <h2>Access denied!</h2>;
   }
 
+  console.log("ingredientsInput", ingredientsInput);
+  console.log("ingredientsSuggestion", ingredientSuggestions);
+
```

Comment:
Are those console-log`s still in your code? If yes, they should be removed before merging into main.